.slather.yml

```diff
@@ -1,5 +1,5 @@
-coverage_service: coveralls
 xcodeproj: ./Networking.xcodeproj
 scheme: Networking
 ignore:
   - NetworkingTests/*
```

Comment:
Shouldn't you add Carthage? I have it but because it was already there, don't know if it's needed.